docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md

```diff
@@ -41,10 +41,31 @@ Following are the [fields](/docs/manage/fields/) which will be created as part o
 
 In this step, you will configure the yaml required for macOS Collection.
 
-1. (Optional) You can add a custom tag, if desired. (There are no parameters required for the macOS app to install.)
-2. Click on the **Download YAML File** button to get the yaml file.
+(Optional) You can add a custom tag, if desired. (There are no parameters required for the macOS app to install.)
 
-<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-YAML.png' style={{border:'1px solid black'}} alt="YAML" />
+#### Enable process metric collection (Optional)
+
+By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).
+
+Click the **Enable process metric collection** checkbox to collect (process level metric)[https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md].
+- **Name of process**. Add the list of process names.
+- **Include/Exclude the above pattern**. Signifies if you want to exclude or include the metrics for the processes listed previously.
+- **Match type for process name**. Select if the process name given should be considered for a strict match with the host machine processes or if it should be considered as regex when matching.
+
+<img src='img/integrations/hosts-operating-systems/process-metric-collection.png' alt="process-metric-collection" width="400"/>
+
+:::note
+If the process list needs to be edited in the future, you can edit it manually in the OTEL config yaml by adding/removing in the names list under process scrapper.
+
+```sh
+process:
+  include:
+    names: [ <process name1>, <process name2> ... ]
+    match_type: <strict|regexp>
+```
+:::
+
+Click on the **Download YAML File** button to get the yaml file.<br/><img src='img/integrations/hosts-operating-systems/Mac-YAML.png' alt="Mac-YAML" width="600"/>
```

Comment:
The image is broken because the tagging doesn't include useBaseUrl. For example:
 <img src={useBaseUrl('img/<your-image-file-path>.png')} alt="<your image description>" style={{border: '1px solid black'}} width="<insert-pixel-number>" />

Reply:
fixed!